index.sh

```diff
@@ -0,0 +1,6 @@
+function test () {
```

Comment:
STEP 3: The index.sh => test() gets executed. It's a simple check. If we can spin up bash and check aws version we know the bash layer still works as expected. And we're confident we can release this layer so other projects can reference it.